.github/workflows/e2e-test.yml

```diff
@@ -78,7 +78,239 @@ jobs:
       - run: sudo systemctl start mysql.service
       - run: WCPAY_DIR="$GITHUB_WORKSPACE" bash ./bin/install-wp-tests.sh woocommerce_test root root localhost $WP_VERSION $WC_VERSION false
       - run: npm run test:e2e-setup
-      - run: npm run test:e2e
+      - run: E2E_GROUP='wcpay' E2E_BRANCH='merchant' npm run test:e2e
+      - uses: actions/upload-artifact@v2
+        with:
+          name: screenshots
+          path: screenshots
+  e2e-wcpay-shopper:
```

Comment:
We could use [composite actions](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) to abstract the common steps in a separate file and then use them in the main one by passing the relevant group and branch variables. This helps us reuse the common steps and avoid duplication.

Reply:
Sure. Will look into it :)

Reply:
@deepakpathania We don't need to use composite actions now. I've removed edge case tests with WP 5.4, WC 4.4.0 & PHP 7.0 and added the test configs to matrix. So we only need only job configuration now.